meta-lmp-base/classes/lmp.bbclass

```diff
@@ -125,10 +125,18 @@ IMAGE_CMD:ota:append () {
 	if [ "${APP_PRELOAD_WITHIN_OE_BUILD}" = "1" ]; then
 		preload_apps
 	fi
+
+	# Split content /var into a separated folder so it can be consumed by WKS separately
+	if [ "${OSTREE_SPLIT_VAR}" = "1" ]; then
+		rm -rf ${OTA_VAR}
+		mv ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var ${OTA_VAR}
+		mkdir -p ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var
+	fi
 }
 OTA_BOOT = "${WORKDIR}/ota-boot"
-do_image_ota[dirs] += "${OTA_BOOT}"
-do_image_ota[cleandirs] += "${OTA_BOOT}"
+OTA_VAR = "${WORKDIR}/ota-var"
```

Comment:
Wouldn't it be more logical and readable to place setting vars above using them?

Reply:
Funny enough most tasks end up defining the variables and cleaning actions after the task itself is defined (if you see the other tasks in this class). We can probably have a separated commit to move things around if needed (here I would prefer to just add the new variable to avoid a larger diff).